README.md

```diff
@@ -218,6 +221,58 @@ if (entity != null) {
 }
 ```
 
+Google Cloud DNS (Alpha)
+----------------------
+- [API Documentation][dns-api]
+- [Official Documentation][cloud-dns-docs]
+
+*Follow the [activation instructions][cloud-dns-activation] to use the Google Cloud DNS API with your project.*
+
+#### Preview
+
+Here are two code snippets showing simple usage examples from within Compute/App Engine. Note that you must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+
+The first snippet shows how to create a zone resource. Complete source code can be found on
+[CreateAndListZones.java](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/CreateAndListZones.java).
```

Comment:
Thinking about it a bit more, I think the organization of the snippet code (the .java files in gcloud-java-examples) could be improved by having one snippet be "CreateZone" (remove listing, mirror this code exactly), another snippet be "CreateDnsRecord" (remove listing, mirror the snippet below exactly), and then make the 3rd snippet (which is currently DeleteZone) match the code in the "Getting Started" section of the gcloud-java-dns.